packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/webview_cookie.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Configuration to use when creating a new [WebViewPlatformController].
+///
+/// The `autoMediaPlaybackPolicy` parameter must not be null.
```

Comment:
This comment is incorrect (probably forgotten to update after copy/paste action).